### PR TITLE
[AND-2530] Mopub mediation settings improvement.

### PR DESCRIPTION
**Pull request by HuFangshuai**
We already have the same in VungleMediationSettings. No need to type more duplicate settings code.
https://github.com/Vungle/mopub-android-mediation/pull/10/files

### CODE REVIEW

File: Vungle/src/main/java/com/mopub/mobileads/VungleRewardedVideo.java [hunk 1]
```diff
@@ -204,11 +203,17 @@ private boolean validateIdsInServerExtras(Map<String, String> serverExtras) {
     }
 
     private void setUpMediationSettingsForRequest(AdConfig adConfig) {
-        final VungleMediationSettings globalMediationSettings =
-                MoPubRewardedVideoManager.getGlobalMediationSettings(VungleMediationSettings.class);
-        final VungleMediationSettings instanceMediationSettings =
-                MoPubRewardedVideoManager.getInstanceMediationSettings(VungleMediationSettings.class, mAdUnitId);
-
+         VungleMediationConfiguration globalMediationSettings =
+                MoPubRewardedVideoManager.getGlobalMediationSettings(VungleMediationConfiguration.class);
```
Comment (HuFangshuai) [inline Vungle/src/main/java/com/mopub/mobileads/VungleRewardedVideo.java:207]: For api `getGlobalMediationSettings` and `getInstanceMediationSettings` MoPub only handle the two classes which must be of exactly equal types.

File: Vungle/src/main/java/com/mopub/mobileads/VungleMediationConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        Object ordinalViewCount = localExtras.get(Builder.EXTRA_ORDINAL_VIEW_COUNT_KEY);
+        if (ordinalViewCount instanceof Integer)
+            adConfig.setOrdinal((Integer) ordinalViewCount);
+        Object adOrientation = localExtras.get(Builder.EXTRA_ORIENTATION_KEY);
+        if (adOrientation instanceof Integer)
+            adConfig.setAdOrientation((Integer) adOrientation);
+    }
+
+    static boolean isStartMutedNotConfigured(Map<String, Object> localExtras) {
+        return !localExtras.containsKey(Builder.EXTRA_START_MUTED_KEY) || !localExtras.containsKey(Builder.EXTRA_SOUND_ENABLED_KEY);
```
Comment (alexander-gubov) [inline Vungle/src/main/java/com/mopub/mobileads/VungleMediationConfiguration.java]: Should be "&&"

**Review by alexander-gubov — CHANGES_REQUESTED** [review]
please check https://github.com/Vungle/mopub-android-mediation/pull/13#discussion_r362496130

**Review by alexander-gubov — APPROVED** [review]
LGTM

**Comment by alexander-gubov** [comment]
have you changed it back?

File: Vungle/src/main/java/com/mopub/mobileads/VungleMediationConfiguration.java (hunk 2 above)
Comment (HuFangshuai) [reply Vungle/src/main/java/com/mopub/mobileads/VungleMediationConfiguration.java]: fixed.

**Pull request merged**